models/src/main/java/com/vimeo/networking2/VideoFile.kt

```diff
@@ -27,6 +26,7 @@ data class VideoFile(
     /**
      * The URL for logging events.
      */
+    @Internal
```

Comment:
are the link and linkExpirationTime accessible? the previous comment was at the class level while this annotation is applied to the property level.

Reply:
There was an issue in the OpenAPI spec. The `VideoFile` object requires a capability. I will annotate the class as internal.